2-RTB/Norme_di_progetto/processiOrganizzativi.tex

```diff
@@ -0,0 +1,150 @@
+\subsection{Attività}
+Il processo di gestione delle infrastrutture tecniche si articola nelle seguenti attività:
+\begin{enumerate}
+    \item istanziazione del processo;
+    \item creazione delle infrastrutture;
+    \item manutenzione delle infrastrutture.
+\end{enumerate}
+\subsubsection{Istanziazione del processo}
+L'attività sia articola nel compito di definire gli obiettivi e i requisiti delle infrastrutture tecniche. Vengono istazniati i protocolli di gestione e si stabiliscono le procedure per il monitoraggio e il controllo delle infrastrutture.
+
+\subsubsection{Creazione delle infrastrutture}
+L'attività sia articola nel compito di progettare e l'implementare le infrastrutture tecniche definite precedentemente. Vengono configurati e si mettono in funzione i componenti hardware e software necessari al supporto delle operazioni.
+
+\subsubsection{Manutenzione delle infrastrutture}
+%La manutenzione delle infrastrutture è un'attività continua e costante. Coinvolge la sorveglianza e la manutenzione preventiva e correttiva delle infrastrutture tecniche. Si assicura che siano operative, sicure e ottimizzate per il carico di lavoro previsto. %
+L'attività si articola nei seguenti compiti:
+            \begin{enumerate}
+                
+            \item Le attività sono suddivise durante le riunioni in task specifiche;
+            \item I ticket relativi a tali task vengono creati su \ccgloss{Jira}, dettagliando gli obiettivi come elementi da completare (\textit{to do});
+            \item Ciascun membro del gruppo si assegna autonomamente una task, indicando lo stato di avanzamento come "in corso" (\textit{in progress});
+            \item Al termine, il membro segnala ai verificatori la necessità di esaminare il lavoro completato, trasferendo la task allo stato di "da riesaminare" (\textit{to revise});
+            \item Un verificatore designato esamina il lavoro eseguito da un altro membro e sposta la task allo stato di "in fase di verifica" (\textit{in checking});
+            \item Dopo la revisione, se l'esito è positivo, il lavoro viene sottoposto all'approvazione del responsabile e la task passa allo stato di "da approvare" (\textit{to approve}). Se la revisione ha esito negativo, la task torna allo stato "da completare" (\textit{to do}), e il processo riprende dal punto 3;
+            \item Una volta che il responsabile approva il contenuto della task, questa viene spostata allo stato di "completata" (\textit{done}), concludendo il ciclo di vita del ticket.
+            \end{enumerate}
+In conclusione, questo processo strutturato, dall'assegnazione delle attività alla loro revisione e approvazione, offre un efficace schema di gestione, assicurando il completamento e la validazione delle task in un ciclo di vita chiaro e definito.
```

Comment:
Formalmente questo sarebbe il processo di una subtask, per giunta una subtask non passa per l'approvazione del responsabile ma una volta passata la verifica andrà in done, il responsabile accetta Task intere quando il loro stato è To Approve, cioè quando tutte le sue sottotask sono finite. BIsogna capire quanto in profondità si vuole andare con la spiegazione